pastastore/store.py

```diff
@@ -1180,18 +1183,19 @@ def add_stressmodel(
 
     def solve_models(
         self,
-        mls: Optional[Union[ps.Model, list, str]] = None,
+        modelnames: Union[List[str], None] = None,
```

Comment:
In a lot of pastastore functions I've allowed single strings as well, which  is converted to [str] internally. I think `_parse_names` does that actually, so that can be added to the type hint as option.